src/OrchardCore.Modules/OrchardCore.Search/OrchardCore.Search.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+  <PropertyGroup>
+    <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
+    <!-- NuGet properties-->
+    <Title>OrchardCore Search</Title>
+    <Description>$(OCCMSDescription)
+
+    Creates a frontend search that can be used with multiple search providers.</Description>
```

Comment:
It has bugged me for a while that we don't have a Search feature for this and it's all in the Lucene module, nice!